utils/staking-miner/src/signer.rs

```diff
@@ -57,9 +57,15 @@ pub(crate) async fn signer_uri_from_string<
 			Hash = Hash,
 		> + EPM::Config,
 >(
-	seed: &str,
+	mut seed_or_path: &str,
 	client: &SharedRpcClient,
 ) -> Result<Signer, Error<T>> {
+	seed_or_path = seed_or_path.trim();
+
+	let seed = match std::fs::read(seed_or_path) {
+		Ok(s) => String::from_utf8(s).map_err(|_| Error::<T>::AccountDoesNotExists)?,
+		Err(_) => seed_or_path.to_string(),
```

Comment:
Is this a common pattern? Overloading like this with something as sensitive as an account seed feels strange. I would prefer to have two separate, explicit flags for a seed as an arg vs a file path arg.

Reply:
I def use and like this patter in my other projects. The problem with raw seed is that you always need to make sure you are executing commands such that they are not saved in your bash history, which makes it hard to use for testing. Contrary, in production you mostly want this. 

https://github.com/paritytech/polkadot-scripts/blob/b88f0ca1e568899e2c6dd2c863a178fdf2342e67/src/helpers/index.ts#L22-L36

will consider two flags, but I think this is also fine.

Reply:
I agree with Zeke two separate flags is better because: i) easier to understand as user and ii) it's makes the code easier to understand and maintain IMO.

Reply:
sorry, for now won't do this. I think the boilerplate needed to make sure one of these two is included is beyond this PR (and generally not worth it). We can revisit this later, in a dedicated PR.